bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/config/VelbusBridgeConfig.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.velbus.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link VelbusBridgeConfig} class represents the configuration of a Velbus Bridge.
+ *
+ * @author Cedric Boon - Initial contribution
+ */
+@NonNullByDefault
+public class VelbusBridgeConfig {
+    public @NonNullByDefault({}) Integer timeUpdateInterval;
+    public @NonNullByDefault({}) Integer reconnectionInterval;
```

Comment:
Better to use primitives:
```suggestion
    public int timeUpdateInterval;
    public int reconnectionInterval;
```

Reply:
Modified in commit d57ba9d